### PR TITLE
Annotate NormOP test to skip grad check

### PR DESCRIPTION
**Pull request by willthefrog**
The function 'check_grad' for large inputs is too slow, however it is desirable to cover the forward implementation.

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_norm_op.py [hunk 1]
```diff
@@ -63,25 +63,25 @@ def init_test_case(self):
         self.epsilon = 1e-8
 
 
+@skip_check_grad_ci(reason="check_grad on large input is too slow")
 class TestNormOp4(TestNormOp):
```
Comment (luotao1) [inline python/paddle/fluid/tests/unittests/test_norm_op.py:68]: 有测过要跑多久么？

Comment (willthefrog) [reply python/paddle/fluid/tests/unittests/test_norm_op.py:68]: 在P40上测了下，跑了一个小时了没动静，考虑到有两个比较大的，至少2个小时以上吧。

Comment (willthefrog) [reply python/paddle/fluid/tests/unittests/test_norm_op.py:68]: 估计是cpu算对照的慢，所以v100也不见得会有区别。

Comment (zhangting2020) [reply python/paddle/fluid/tests/unittests/test_norm_op.py:68]: 这个单测目的是测试大尺度的输入，PR中描述：为了覆盖CUDAKernel的前向实现，我认为有点不准确，单测中会同时测试CPU和CUDA实现。PR描述和reason都可以完善下，例如：The function 'check_grad' for large inputs is too slow, however the test case is desirable to cover the forward implementation.

Comment (willthefrog) [reply python/paddle/fluid/tests/unittests/test_norm_op.py:68]: Done.

Comment (zhangting2020) [reply python/paddle/fluid/tests/unittests/test_norm_op.py:68]: 还有两个问题：
- continuous-integration/travis-ci/pr未过，请修改。
image
- reason描述的内容：however it is desirable to cover the forward pass这句看起来不太通顺，建议修改为：The function 'check_grad' for large inputs is too slow, however the test case is desirable to cover the forward implementation.

Comment (willthefrog) [reply python/paddle/fluid/tests/unittests/test_norm_op.py:68]: 1. done.
2. the phrasing is fine, would not say the same about the suggested one though.

**Review by zhangting2020 — APPROVED** [review]

**Pull request merged**